src/forms/reset-password-popup/reset-password/index.jsx

```diff
@@ -77,7 +77,9 @@ const ResetPasswordPage = () => {
   useEffect(() => {
     if (formErrors && Object.keys(formErrors).length > 0 && errorRef.current) {
```

Comment:
This condition will not work because the key will be added in the `formErrors`  state with '' value on focus. And on focus will be called on field focus.
[Reference](https://github.com/edx/frontend-component-authn-edx/blob/63c5e85806125f3bb138138ad02f05af6f3250a8/src/forms/reset-password-popup/reset-password/index.jsx#L73)

Reply:
this is not relevant to my current ticket. I will leave it as it is and will fix it as a bug